Plugins~/Src/MeshSyncClientBlender/msblenContext.cpp

```diff
@@ -647,11 +647,6 @@ ms::LightPtr msblenContext::exportLight(msblenContextState& state, msblenContext
 
 ms::MeshPtr msblenContext::exportMesh(msblenContextState& state, msblenContextPathProvider& paths, BlenderSyncSettings& settings, const Object *src)
 {
-    // ignore particles
-    if (//FindModifier(src, eModifierType_ParticleSystem) ||
-        FindModifier(src, eModifierType_ParticleInstance) )
-        return nullptr;
-
```

Comment:
Wonder why this existed in the first place.. Maybe we could have a toggle to sync particle systems like the Bake Modifiers option, to avoid disrupting existing workflows

Reply:
For reference: it seems that this check was added because it required further implementation
https://github.com/unity3d-jp/MeshSync/commit/96c10275bc4f4325c1483f1a7bf2cfffc2e7d132

So, are you saying that we can sync particle animations now by just removing these lines ?

Reply:
I don't, at least. I thought you did.
How did you test your change then ?